code/modules/mob/living/carbon/xenomorph/abilities/lurker/lurker_powers.dm

```diff
@@ -212,6 +212,14 @@
 				to_chat(xeno, SPAN_WARNING("There's something blocking us from striking!"))
 				return
 
+	for(var/mob/living/carbon/carbonara in get_turf(targeted_atom))
+		hit_target = carbonara
+		to_chat(xeno, SPAN_XENODANGER("We attack [hit_target] with our tail, throwing it back after stabbing it with our tail!"))
+
+	if(targeted_atom == hit_target) //reward for a direct hit
+		to_chat(xeno, SPAN_XENOHIGHDANGER("We directly slam [hit_target] with our tail, throwing it back after impaling it on our tail!"))
+		hit_target.apply_armoured_damage(15, ARMOR_MELEE, BRUTE, "chest")
```

Comment:
Move the `to_chat(xeno, SPAN_XENODANGER("We attack [hit_target] with our tail, throwing it back after stabbing it with our tail!"))` to be the else case for `if(targeted_atom == hit_target)`

In other words; you only send one message that is either the direct hit message, or the not direct hit messaage.

Reply:
tested to make it this way but it really becomes a lot of extra code with no real benefit